--- a/Test/runtests
+++ b/Test/runtests
@@ -7,8 +7,8 @@
 
 TARGETDIR=${1:-localResults}
 BASEDIR=baseResults
-EXE=${2:-../build/install/bin/glslangValidator}
-REMAPEXE=${3:-../build/install/bin/spirv-remap}
+EXE=${2:-../build/StandAlone/glslangValidator}
+REMAPEXE=${3:-../build/StandAlone/spirv-remap}
 HASERROR=0
 mkdir -p $TARGETDIR
 

--- glslang-715c353a15836e5ae192a64a4cf54e2ce7e8d66a/Test/glslangValidator
+++ glslang-715c353a15836e5ae192a64a4cf54e2ce7e8d66a-new/Test/glslangValidator
@@ -1,2 +1,2 @@
 #!/usr/bin/env bash
-../build/install/bin/glslangValidator $*
+../build/StandAlone/glslangValidator $*
