From: Jan Engelhardt <jengelh@inai.de>
Date: 2017-08-07 20:22:29.456870387 +0200

---
 StandAlone/spirv-remap.cpp |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: glslang/StandAlone/spirv-remap.cpp
===================================================================
--- a/StandAlone/spirv-remap.cpp
+++ b/StandAlone/spirv-remap.cpp
@@ -227,7 +227,7 @@ namespace {
                 }
             }
             else if (arg == "--version" || arg == "-V") {
-                std::cout << basename(argv[0]) << " version 0.97 " << __DATE__ << " " << __TIME__ << std::endl;
+                std::cout << basename(argv[0]) << " version 0.97 " << std::endl;
                 exit(0);
             } else if (arg == "--input" || arg == "-i") {
                 // Collect input files
