--- CMakeLists.txt	2019-11-26 20:19:20.967645516 +0200
+++ CMakeLists.txt	2019-11-26 20:21:29.548358547 +0200
@@ -146,7 +146,7 @@ endif()
 
 # pkg-config file
 configure_file( libclc.pc.in libclc.pc @ONLY )
-install( FILES ${CMAKE_CURRENT_BINARY_DIR}/libclc.pc DESTINATION ${CMAKE_INSTALL_DATADIR}/pkgconfig )
+install( FILES ${CMAKE_CURRENT_BINARY_DIR}/libclc.pc DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig )
 install( DIRECTORY generic/include/clc DESTINATION ${CMAKE_INSTALL_INCLUDEDIR} )
 
 if( ENABLE_RUNTIME_SUBNORMAL )
@@ -155,7 +155,7 @@ if( ENABLE_RUNTIME_SUBNORMAL )
 	add_library( subnormal_disable STATIC
 		generic/lib/subnormal_disable.ll )
 	install( TARGETS subnormal_use_default subnormal_disable ARCHIVE
-		DESTINATION ${CMAKE_INSTALL_DATADIR}/clc )
+		DESTINATION ${CMAKE_INSTALL_LIBDIR}/clc )
 endif()
 
 find_program( PYTHON python )
@@ -275,7 +275,7 @@ foreach( t ${LIBCLC_TARGETS_TO_BUILD} )
 				            prepare_builtins )
 		add_custom_target( "prepare-${obj_suffix}" ALL
 		                   DEPENDS "${obj_suffix}" )
-		install( FILES ${CMAKE_CURRENT_BINARY_DIR}/${obj_suffix} DESTINATION ${CMAKE_INSTALL_DATADIR}/clc )
+		install( FILES ${CMAKE_CURRENT_BINARY_DIR}/${obj_suffix} DESTINATION ${CMAKE_INSTALL_LIBDIR}/clc )
 		# nvptx-- targets don't include workitem builtins
 		if( NOT ${t} MATCHES ".*ptx.*--$" )
 			add_test( NAME external-calls-${obj_suffix}
@@ -293,7 +293,7 @@ foreach( t ${LIBCLC_TARGETS_TO_BUILD} )
 					   create_symlink ${obj_suffix}
 					   ${alias_suffix}
 			                   DEPENDS "prepare-${obj_suffix}" )
-			install( FILES ${CMAKE_CURRENT_BINARY_DIR}/${alias_suffix} DESTINATION ${CMAKE_INSTALL_DATADIR}/clc )
+			install( FILES ${CMAKE_CURRENT_BINARY_DIR}/${alias_suffix} DESTINATION ${CMAKE_INSTALL_LIBDIR}/clc )
 		endforeach( a )
 	endforeach( d )
 endforeach( t )
