From 201cacaa74446657d884786d31411a98f6f9c17e Mon Sep 17 00:00:00 2001
From: serge-sans-paille <sguelton@redhat.com>
Date: Mon, 4 Feb 2019 21:25:45 +0000
Subject: [PATCH] Workaround GCC 9 bug when handling bitfields.

---
 include/clang/Frontend/TextDiagnosticPrinter.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/clang/Frontend/TextDiagnosticPrinter.h b/include/clang/Frontend/TextDiagnosticPrinter.h
index 3cb4e02edf..d47d71930c 100644
--- a/include/clang/Frontend/TextDiagnosticPrinter.h
+++ b/include/clang/Frontend/TextDiagnosticPrinter.h
@@ -35,7 +35,7 @@ class TextDiagnosticPrinter : public DiagnosticConsumer {
   /// A string to prefix to error messages.
   std::string Prefix;
 
-  unsigned OwnsOutputStream : 1;
+  bool OwnsOutputStream;
 
 public:
   TextDiagnosticPrinter(raw_ostream &os, DiagnosticOptions *diags,
-- 
2.19.2

 
