diff -up SPIRV-Tools-3e4abc9ac382b6fc7a88316a700652404b6fb9f7/CMakeLists.txt.dma SPIRV-Tools-3e4abc9ac382b6fc7a88316a700652404b6fb9f7/CMakeLists.txt
--- SPIRV-Tools-3e4abc9ac382b6fc7a88316a700652404b6fb9f7/CMakeLists.txt.dma	2019-11-12 13:58:34.347573658 +1000
+++ SPIRV-Tools-3e4abc9ac382b6fc7a88316a700652404b6fb9f7/CMakeLists.txt	2019-11-12 13:58:47.166919110 +1000
@@ -215,7 +215,7 @@ if(ENABLE_SPIRV_TOOLS_INSTALL)
     endmacro()
   else()
     macro(spvtools_config_package_dir TARGET PATH)
-      set(${PATH} lib/cmake/${TARGET})
+      set(${PATH} ${CMAKE_INSTALL_LIBDIR}/cmake/${TARGET})
     endmacro()
   endif()
 
